inspirehep/modules/authors/tasks.py

```diff
@@ -27,16 +27,14 @@
 import os
 
 from flask import current_app, url_for
-from sqlalchemy.orm.exc import NoResultFound
-
+from inspire_dojson.utils import strip_empty_values
+from inspire_schemas.api import validate
 from invenio_accounts.models import User
 from invenio_oauthclient.models import UserIdentity
+from sqlalchemy.orm.exc import NoResultFound
 
-from inspire_dojson.utils import strip_empty_values
-from inspire_schemas.api import validate
 from inspirehep.modules.forms.utils import filter_empty_elements
-from inspirehep.modules.workflows.utils import with_debug_logging
-
+from inspirehep.modules.workflows.utils.utils import with_debug_logging
```

Comment:
To avoid this ugly `utils.utils`, just put the stuff that was in `utils.py` before in `utils/__init__.py`.

Reply:
Thx, this confused me more than you think!